### DISCUSSION TITLE
After restarting the printer mesh leveling settings are resetting.

### DISCUSSION THREAD

**Discussion by BurakFati** [mriscoc/Ender3V2S1#1231] [Q&A] [answered]
In last 2 releases i have a issue about mesh leveling settings, it keeps resetting when i restart the printer. Is there a way to save it and restore it when a gcode called?

*only mesh points X and mesh points Y resetting

**Comment by mriscoc** [comment]
You must save the mesh settings with G29 Sn (Where n is the mesh slot number) or by using `Advanced / Mesh Leveling / Save Bed Mesh`

**Reply by BurakFati** [reply]
Thanks it helped me out for mesh points X and mesh points Y saving but mesh inset are still resetting to Maximize Area.

**Reply by mriscoc** [reply]
What is the name/version of the firmware that you are using?

**Reply by BurakFati** [reply]
It's **Ender3V2-422-BLTUBL-MPC-20240125.bin** currently.

**Reply by mriscoc** [reply, marked as answer]
Follow this process:
Set the mesh slot, density and inset, run a auto mesh leveling or reset the mesh, save the mesh data, save settings.
The next time the printer starts, all mesh settings and parameters will be automatically loaded.
